fraud-detection-python/cloud-functions/geocode-addresses/main.py

```diff
@@ -0,0 +1,108 @@
+import base64
+import json
+import os
+import requests
+from urllib.parse import urlencode
+from google.cloud import bigquery
+
+dataset_name = 'invoice_parser_results'
+table_name = 'geocode_details'
+my_schema = [
```

Comment:
can you not use the contents of geocode_details.json instead of this since they're basically the same ?

Reply:
Yes, it probably should. I didn't touch any of this code from the previous author.